app/models/manageiq/providers/ibm_power_hmc/infra_manager/event_target_parser.rb

```diff
@@ -78,6 +78,8 @@ def handle_usertask(usertask)
       else
         []
       end
+    else
+      []
```

Comment:
We tend to not like multiple nested conditionals like this if we can avoid it.

I can see a few ways to go about this:
```
return [] unless usertask["status"].eql?("Completed")

case ...
```

Or
```
result = if usertask["status"].eql?("Completed")
  case ...
  end

result || []
```

Reply:
Fixed